@plotly/webpack-dash-dynamic-import/src/index.js

```diff
@@ -39,25 +43,27 @@ Object.defineProperty(__webpack_require__, 'p', {
     })()
 });
 
-const __jsonpScriptSrc__ = jsonpScriptSrc;
-jsonpScriptSrc = function(chunkId) {
-    let script = getCurrentScript();
-    let isLocal = isLocalScript(script);
+if (typeof jsonpScriptSrc !== 'undefined') {
```

Comment:
`jsonpScriptSrc` is not define is there are no async resources, the `typeof ... === 'undefined'` check is the only way in JS to test for the existence/non-existence of a variable in a safe way.